docs/deployer.md

```diff
@@ -1,5 +1,10 @@
 # Deployer
 
-The deployer image is a Docker image that GCP will run in the customer Kubernetes cluster to the app's configuration to the Kubernetes API when users deploy the app from Cloud console.
+The `deployer` image is a Docker image that GCP will run in the customer GKE cluster to create all needed resources.
```

Comment:
Are we planning to bring this into the main “README.md” file? For example see: https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/k8s/wordpress. If that wasn’t the current plan I would highly suggest that we should bring all the different doc files into a single README.md itself rather than all these separate .md files.

Do we support Enthos as well? It would be good to clarify this in our document.

Can we add a simple diagram showing how the Elastic Agent is deployed and running in a very simple GKE cluster? It would be good for users to know if Elastic Agent runs as a DaemonSet etc. information?

Reply:
> Are we planning to bring this into the main “README.md” file?

I wrote this documentation as contributors documentation, not end user documentation. A huge README makes contributing more difficult. I agree that for end users we should provide a more cohesive experience, I will discuss it with the docs team!

> Do we support Enthos as well? It would be good to clarify this in our document.

Yes, but I've not tested it. I will add it to the documentation.
(For future reference requirements are [here](https://cloud.google.com/marketplace/docs/partners/kubernetes/create-app-package#on-prem-reqs))

> Can we add a simple diagram showing how the Elastic Agent is deployed and running in a very simple GKE cluster? It would be good for users to know if Elastic Agent runs as a DaemonSet etc. information?

What about adding this information to the Elastic Agent on Kubernetes documentation? Given that the deployment structure is the same, I think would be better to have it in the general documentation than here as a special case.

Reply:
updated in https://github.com/elastic/elastic-agent-gcp-kubernetes-app/pull/1/commits/de3efcdea01e14f062af2ac990c9bb4a184820d6